### PR TITLE
added vlan fields again

### PR DESCRIPTION
**Pull request by usmanazFIU**

### CODE REVIEW

File: modules/circuits/views/nodes/nodes/nodes.php [hunk 1]
```diff
@@ -90,12 +90,12 @@
     </select>
   </div>
 
-<!--
+
   <div class="form-group">
     <label for="exampleInputPassword1">Source VLAN</label>
     <input type="number" maxlength="4" class="form-control" id="source_vlan" name="source_vlan" placeholder="0-4096" >
```
Comment (italovalcy) [inline modules/circuits/views/nodes/nodes/nodes.php:96]: @usmanazFIU can you please change the placeholder here to inform VLAN id will be from 1-4095 ? (0 and 4096 should NOT be considered valid values according to the Datamodel spec https://docs.google.com/document/d/1lgxjIT144EFu1G_OVcU19hN1cSUT_v2-tE0Z-7UlkNg/edit#bookmark=id.vixmudxbm9j7)

File: modules/circuits/views/nodes/nodes/nodes.php [hunk 2]
```diff
@@ -279,20 +279,20 @@ function(c) {
...
+    var source_vlan=$('#source_vlan').val();
+    var destination_vlan=$('#destination_vlan').val();
     var latency_required=$('#latency_required').val();
     var bandwidth_required=$('#bandwidth_required').val();
 
-    //if(source_vlan>4096||source_vlan<0){
-    //  alert("source vlan should be between 0-4096");
-    //  return;
-    //}
+    if(source_vlan>4096||source_vlan<0){
```
Comment (italovalcy) [inline modules/circuits/views/nodes/nodes/nodes.php:287]: same thing on the vlan range here, vlan 4096 and 0 should not be considered valid values

File: modules/circuits/views/nodes/nodes/nodes.php [hunk 3]
```diff
@@ -279,20 +279,20 @@ function(c) {
...
+     alert("source vlan should be between 0-4096");
```
Comment (italovalcy) [inline modules/circuits/views/nodes/nodes/nodes.php:288]: message should also reflect the valid values

File: modules/circuits/views/nodes/nodes/nodes.php [hunk 4]
```diff
@@ -279,20 +279,20 @@ function(c) {
...
+     return;
+    }
 
-    //else if(destination_vlan>4096||destination_vlan<0){
-    //  alert("destination vlan should be between 0-4096");
-    //  return;
-    //}
+    else if(destination_vlan>4096||destination_vlan<0){
+     alert("destination vlan should be between 0-4096");
```
Comment (italovalcy) [inline modules/circuits/views/nodes/nodes/nodes.php:293]:
```suggestion
    else if(destination_vlan>=4096||destination_vlan<=0){
     alert("destination vlan should be between 1-4095");
```

**Review by italovalcy — CHANGES_REQUESTED** [review]
Excellent to have the VLAN IDs back to the game, @usmanazFIU. Very nice done! I just left a few comments regarding the VLAN range. Also, is SDX-Controller already supporting user requested VLANs? we removed this field back in Feb because SDX-Controller didn't support.

File: modules/circuits/views/nodes/nodes/nodes.php (hunk 1 above)
Comment (usmanazFIU) [reply modules/circuits/views/nodes/nodes/nodes.php:96]: sure. let me change it

**Review by italovalcy — APPROVED** [review]
LGTM

**Pull request closed without merging**